pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -363,6 +363,28 @@ where
         Self::prepare_handle(module, 0)
     }
 
+    pub fn gr_exec_settings(r: u32) -> Result<Exec<T>, &'static str> {
+        let repetitions = r * API_BENCHMARK_BATCH_SIZE;
+        let settings_offset = COMMON_OFFSET;
+
+        let module = ModuleDefinition {
+            memory: Some(ImportedMemory::new(SMALL_MEM_SIZE)),
+            imported_functions: vec![SysCallName::ExecSettings],
+            handle_body: Some(body::syscall(
+                repetitions,
+                &[
+                    // offset where to write settings
+                    InstrI32Const(settings_offset),
+                    // version. TODO: Should it be benched based on version?
```

Comment:
Should this be benched based on version? I.e. should we have different benchmarks for different version?

Reply:
I'd do different version, since this call will be used by pretty much every single program